code/game/machinery/computer/security_alert.dm

```diff
@@ -47,6 +47,7 @@ TODO: literally every alarm but SPS alarms.
 /obj/machinery/computer/security_alerts/update_icon(var/showalert = FALSE)
 	..()
 	if(stat & (NOPOWER|BROKEN))
+		overlays = 0
```

Comment:
overlays.Cut() is probably preferable

Reply:
How so? It's the same code as turning the overlays off normally.

Reply:
This way overlays would be a list with no elements instead of the number 0. I suspect this is more safe. That said, you are correct in that there are plenty of examples of using `overlays = 0` in the code presently.

Reply:
`overlays.len = 0` is probably what they meant to write, which is equivalent to `overlays.Cut()`